app/views/songs/_plist_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_for(:playlist, url: "/add", method: :patch) do |f| %>
+  <%= f.select(:id, current_user.playlists.map{ |n| [n.name, n.id] }, class: 'playlist-dropdown') %>
+  <%= f.label :song_id, class: 'hidden' %>
```

Comment:
Is this necessary as a hidden form value, or would it be better as a RESTful route parameter somehow?